src/vs/workbench/test/common/notifications.test.ts

```diff
@@ -15,14 +15,14 @@ suite('Notifications', () => {
 
 		// Invalid
 		assert.ok(!NotificationViewItem.create({ severity: Severity.Error, message: '' }));
-		assert.ok(!NotificationViewItem.create({ severity: Severity.Error, message: null }));
+		assert.ok(!NotificationViewItem.create({ severity: Severity.Error, message: null! }));
```

Comment:
@usernamehw what does this mean? Are you asserting that `null` is not null?

Reply:
Specifically, what does `null!` mean? I thought that would mean to the compiler that `null` cannot be null, which doesn't make sense to me.

Reply:
I learned it yesterday myself. Yes, that's exactly what it means.

Those tests are bound to be failed. null is strict checked and cannot be passed as message, but there is a need to have a failing tests without thousands of compile errors in project. (And preferably by not casting everything to `any` to not loose autocomplete).